framework/wazuh/tests/test_utils.py

```diff
@@ -123,8 +125,8 @@ def test_execute(mock_output):
 
 
 @pytest.mark.parametrize('error_effect, expected_exception', [
-    (CalledProcessError(returncode=10000, cmd='Unspected error', output='{"data":"Some data", "message":"Error", '
-                                                                        '"error":10000}'), 10000),
+    (CalledProcessError(returncode=1000, cmd='Unspected error', output='{"data":"Some data", "message":"Error", '
+                                                                       '"error":1000}'), 1000),
     (Exception, 1002),
     (CalledProcessError(returncode=1, cmd='Unspected error', output={}), 1003),
     (CalledProcessError(returncode=1, cmd='Unspected error', output='{"error":10000}'), 1004),
```

Comment:
```suggestion
    (CalledProcessError(returncode=1, cmd='Unexpected error', output={}), 1003),
    (CalledProcessError(returncode=1, cmd='Unexpectederror', output='{"error":10000}'), 1004),
```
Should `{"error":10000}` be `{"error":1000}`?